content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
+All resource types are either scoped by the cluster (`/apis/<GROUP>/<VERSION>/*`) or to a namespace (`/apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/*`). A namespace-scoped resource type will be deleted when its namespace is being deleted, and access to a namespaced resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
 
 * Cluster-scoped resources:
 
-  * `GET /apis/GROUP/VERSION/RESOURCETYPE` - return the collection of resources of the resource type
-  * `GET /apis/GROUP/VERSION/RESOURCETYPE/NAME` - return the resource with NAME under the resource type
+  * `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>` - return the collection of resources of the resource type
+  * `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>/<NAME` - return the resource with NAME under the resource type
 
 * Namespace-scoped resources:
 
-  * `GET /apis/GROUP/VERSION/RESOURCETYPE` - return the collection of all instances of the resource type across all namespaces
-  * `GET /apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE` - return collection of all instances of the resource type in NAMESPACE
-  * `GET /apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE/NAME` - return the instance of the resource type with NAME in NAMESPACE
+  * `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>` - return the collection of all instances of the resource type across all namespaces
+  * `GET /apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/<RESOURCETYPE>` - return collection of all instances of the resource type in NAMESPACE
+  * `GET /apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/<RESOURCETYPE>/<NAME>` - return the instance of the resource type with NAME in NAMESPACE
 
-Since a namespace is a cluster-scoped resource type, you can retrieve the list of all namespaces with `GET /api/v1/namespaces` and details about a particular namespace with `GET /api/v1/namespaces/NAME`.
+Since a namespace is a cluster-scoped resource type, you can retrieve the list of all namespaces with `GET /api/v1/namespaces` and details about a particular namespace with `GET /api/v1/namespaces/<NAME>`.
 
-Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH, and DELETE. Kubernetes uses the term **list** to describe returning a collection of resources to distinguish from retrieving a single resource which is usually called a **get**.
+Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH, and DELETE. Kubernetes uses the term **List** to describe returning a collection of resources to distinguish from retrieving a single resource which is usually called a **Get**.
```

Comment:
Do NOT change `list` to `List` or `get` to `Get` to cause more confusion, please.

Reply:
There are some reference for mapping verbs. Let's be careful when introducing this kind of spelling changes.
https://kubernetes.io/docs/reference/access-authn-authz/authorization/#determine-the-request-verb

Reply:
I think I'd got confused between `watch` (the verb) and `Watch` (the noun). It's hard to document; I'm learning as I go to be honest.

Reply:
I've revised my approach here (having learned the API enough to understand what this page is trying to say).